kubeflow/gcp/create_ca.sh

```diff
@@ -91,6 +91,15 @@ kubectl create secret generic ${secret} \
         --dry-run -o yaml |
     kubectl -n ${namespace} apply -f -
 
+# restart the webhook server if it already exists, ignore the error otherwise
+# this is for https://github.com/kubeflow/kubeflow/issues/3227. 
+# Webhook pod once created loads the secret in the begining and starts serving. 
+# Therefore, if secret is updated, then Webhook pod needs to be restarted  
+webhookPod=$(kubectl get pods -n ${namespace} | grep gcp-cred-webhook- |awk '{print $1;}')
```

Comment:
what happens if grep matches nothing (pod not exist)?
Will that be error or webhookPod just equals empty string?
Do you need to check webhookpod empty before kubectl delete pod?

Reply:
The following line just ignores the error.